### DISCUSSION TITLE
How does tea work with header-only C or C++ libraries?

### DISCUSSION THREAD

**Discussion by mfts** [pkgxdev/pkgx#318] [Help] [answered]
asked via Discord by e.dant

How does tea work with header-only C or C++ libraries? Any way to tell conan, cmake or vcpkg about the headers?
Perhaps tea has an include directory similar to Homebrew's /opt/homebrew/include where the headers would go?

**Comment by mxcl** [comment, marked as answer]
We add to `CMAKE_PREFIX_PATH` for all pkgs. Make knows what to do. For everything else we add to `CPATH`.

You can trivially see this by doing a `tea +pkg --dry-run` to see the environments that are created.
